README.md

```diff
@@ -248,6 +248,10 @@ For Vert.x version 2 check [this page](./vert-x2.md).
 * [Vert.x feeds](https://github.com/aesteve/vertx-feeds) - Example of an RSS aggregator built using Vert.x, Gradle, MongoDB, Redis, Handlebars templates, AngularJS, the event bus and SockJS.
 * [Vert.x Markdown service](https://github.com/aesteve/vertx-markdown-service) - Example on how to use [service-proxy](https://github.com/vert-x3/vertx-service-proxy) with Gradle.
 * [Example using event bus and service proxies to connect vertx and node](https://github.com/advantageous/vertx-node-ec2-eventbus-example) - Step by step example with wiki description showing how to connect Vert.x and Node using event bus and service proxies.
+* [Vert.x Todo-Backend implementation](https://github.com/aesteve/todo-backend-vertx). Pure Java 8 implementation of the Todo MVC backend. Uses a Vert.x LocalMap for storage.
+* [Kotlin Todo-Backend implementation](https://github.com/aesteve/vertx-kotlin-todomvc). Kotlin implementation of the Todo MVC backend.
+* [Scala Todo-Backend implementation](https://github.com/aesteve/vertx-scala-todomvc) Scala implementation of the Todo MVC backend.
+* [Grooveex Todo-Backend implementation](https://github.com/aesteve/todo-backend-grooveex) : Todo MVC backend implementation with Vert.x + Groovy + some syntaxic sugar + DSL routing facilities
```

Comment:
syntaxic should be syntactic. Please add a dot at the end of the description. Replace the colon ':' by a hyphen '-'. And while you are at it do so for the other entries as well (i.e. hyphen between link and description). Thanks!